node/Cargo.toml

```diff
@@ -24,60 +25,61 @@ mainnet-runtime = {path = "../runtime/mainnet"}
 runtime-common = {path = "../runtime/common"}
```

Comment:
Yes, I ran `cargo update` prior to the migration to ensure the latest dependencies are used. Isn't it a good practice?

Reply:
Why ? This template is based on the Substrate Parachain Template, we should use the same dependencies versions.

Reply:
I wasn't aware of that. I simply thought it'd be good to keep up to date with the latest versions, specially for security reasons. In any case I'll undo de changes in `Cargo.lock`.

Now, regardless of that, and as a side note: why is that such a strong constraint? From my perspective:
- Templates are different, and they never merge code, so it doesn't really affect whether use different versions in different templates or not.
-  How do we then track whenever the `Substrate Parachain Template` is updating its dependencies? Because if we do not effectively keep them conveniently track, I sincerely prefer to keep them at least updated with the (likely) newest security features.

Reply:
Fixed in [1c74d00](https://github.com/paritytech/extended-parachain-template/pull/54/commits/1c74d008b375d31f5677f61f93e0986f9a817b20).